src/Expr.cc

```diff
@@ -4887,6 +4929,7 @@ void ListExpr::Assign(Frame* f, Val* v)
 		exprs[i]->Assign(f, (*lv->Vals())[i]->Ref());
 
 	Unref(lv);
+	Unref(v);
```

Comment:
This was a problematic unref because it was already unref'ing via `lv` (the line directly above).  For clarity, going to change it to unref via `v` instead of `lv`, though.